### PR TITLE
add crop Face to jpeg ( cropFaceToJpeg ) method.

### PR DESCRIPTION
**Pull request by curioustushar**
add method crop Face to jpeg  which return image with only face ( jpeg format )
Source :- http://www.codediesel.com/algorithms/face-detection-in-images-using-php/

### CODE REVIEW

File: FaceDetector.php [hunk 1]
```diff
@@ -156,6 +156,13 @@ public function getFace()
         return $this->face;
     }
 
+    public function cropFaceToJpeg() {
+        $canvas = imagecreatetruecolor($this->face['w'], $this->face['w']);
+        imagecopy($canvas, $this->canvas, 0, 0, $this->face['x'], $this->face['x'], $this->face['w'], $this->face['w']);
```
Comment (alex-topface) [inline FaceDetector.php]: probably it should be 
```
imagecopy($canvas, $this->canvas, 0, 0, $this->face['x'], $this->face['y'], $this->face['w'], $this->face['w']);
```

**Comment by alex-topface** [comment]
Hello guys! I'm interested in having this #23 merged in. Is there any possibility to have this done?
@mauricesvay @cse-tushar

**Comment by mauricesvay** [comment]
Hi @alex-topface, I'm not sure this function should be the responsibility of the face detection class. Is there a specific reason for adding it?

**Comment by alex-topface** [comment]
Hi Maurice, 
why do you think it shouldn't? It already has functions `getFace` to detect face and `toJpeg` to print the red border over the face. 
In my project I'd like to compare faces. So I use this class for detecting them. But even if I manage to get face coordinates, I can't extract the face, becouse the `canvas` variable has the `protected` access.

As for me the best solution is to use the code that can extract the face to another file. I can place the new  pull request that fixes the error I posted above and has the possibility to print the result (extracted face) eigther to std out, as it done for now, or to specified file.

Will you be able to accept such PR and create a new release tag?

Thanks, Alex.

**Comment by mauricesvay** [comment]
Alright, sounds ok. I'll merge and create a new release when you update the PR.

**Comment by alex-topface** [comment]
Thanks!

**Comment by mauricesvay** [comment]
Fixed by #25

**Pull request closed without merging**